pom.xml

```diff
@@ -3369,6 +3369,7 @@
                   <arg>-Wconf:msg=Auto-application to \`\(\)\` is deprecated:s</arg>
                   <arg>-Wconf:msg=method with a single empty parameter list overrides method without any parameter list:s</arg>
                   <arg>-Wconf:msg=method without a parameter list overrides a method with a single empty one:s</arg>
+                  <arg>-Wconf:msg=symbol literal is deprecated:s</arg>
```

Comment:
Can we have any feedback before we ignoring the symbol deprecations?

Reply:
@HyukjinKwon I think this may be a compromise for https://github.com/apache/spark/pull/31569, if we don't plan to fix it recently. I just asked in https://github.com/apache/spark/pull/31569, there is no other feedback for the time being except for @cloud-fan.

Reply:
I think this is the only choice for now as we don't want to remove the symbol usage in Spark. We can rely on the Scala feature import and ask the Scala community to keep it forever. If it's removed in the future, we can change the Spark code at that time.

Reply:
Okay, if we're going to keep them explicitly, I'm fine.

Reply:
there are new updates in #31569  ...

Reply:
Wait, I'm still not clear why we can't replace `'foo` with `Symbol(foo)`. That does not remove usage of Symbol. It just removes usage of the deprecated literal syntax. I thought the resolution was that this much was fine? the other issue is kind of conflating the deprecated syntax and Symbol itself.

Reply:
We use symbol literal in our testing DSL because it's easy to write. If we need to write `Symbol("foo")` from now on, I'd vote for updating the test DSL and pick a different syntax, such as https://github.com/apache/spark/pull/31601

Reply:
Yeah it's not as compact. I'm OK with just suppressing it too, if the code churn is too much of a PITA.